src/api/app/views/webui2/webui/project/_monitor_control.html.haml

```diff
@@ -44,6 +44,12 @@
                 = check_box_tag valid_xml_id('repo_' + repository), 1, repository_filter.include?(repository), class: 'custom-control-input'
                 %label.custom-control-label{ for: valid_xml_id('repo_' + repository) }
                   = repository
+      %span#project-monitor-repositories-last-build
+        %button.btn.btn-outline-secondary
+          = check_box_tag 'lastbuild', 1, lastbuild_switch
+          Last time results
```

Comment:
:thought_balloon: 

```
<span title="">Include the results from the last build</span>
```

Reply:
@hennevogel do you want to switch the "last time results" by "Include the results from the last build" ? I'm wondering if it's not too long?

Reply:
I think he wants you to add a title, so when you do mouseover you can read the long text.

Reply:
Oops or not! That's what I understood :thinking:

Reply:
> @hennevogel do you want to switch the "last time results" by "Include the results from the last build" ? I'm wondering if it's not too long?

@vpereira, I think he is telling you to add `title='Include the results from the last build'`

Reply:
yeah, some long for so you can actually understand what this does. What does it do BTW? ;-)

Reply:
![Screenshot_2019-07-02_14-23-06](https://user-images.githubusercontent.com/37418/60512627-5a644a80-9cd5-11e9-8b05-c9ec649df102.png)